docs/demo.mdx

```diff
@@ -0,0 +1,370 @@
+---
+title: Internal Demo Page for Styling
+description: This page is for internal use only.
+---
+
+<head>
+  <meta name="robots" content="noindex, nofollow" />
+  <title>Detox Interal Demo Page</title>
+</head>
+
+import HomepageHeader from '@site/src/components/Homepage/Header/HomepageHeader';
+
+import '@site/src/css/custom.css';
```

Comment:
Why do we import CSS here?

Reply:
Hmm. Thought I needed it for the admonitions stylings but it's not necessary after all. Removed. Thanks.